### PR TITLE
Documentation for contributors

### LINKED ISSUE #1164: Documentation for contributors
We should have a page on the docs.ferretdb.io that explains contributing in general and links to CONTRIBUTING.md in repos:
* FerretDB
* github-actions
* dance

### PR DESCRIPTION
**Pull request by Fashander**
# Description

This PR closes #1164 .

## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# How to contribute to FerretDB
+
+FerretDB is an open source project and everyone – developers and non-developers – is welcome to contribute.
+If you’re interested in contributing to FerretDB projects, this documentation will help you get started.
+
+Here are some of the current FerretDB projects that you can fork and contribute to:
+
+* [FerretDB](https://github.com/FerretDB/FerretDB): This repository contains the FerretDB project – an open source alternative project to MongoDB.
+* [github-actions](https://github.com/FerretDB/github-actions): This repository houses our shared GitHub Actions for FerretDB, dance, and other repositories.
+* [dance (stub)](https://github.com/FerretDB/dance): This repository is a FerretDB integration testing tool.
```
Comment (AlekSi) [inline website/docs/contributing.md:14]: Dance is not a stub :) Not sure why I wrote that.

Also, I think we should move it above github-actions

File: website/docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+For those taking their first steps in contributing to an open source project.
+Please take a look at this post on [how to contribute to open source software](https://www.ferretdb.io/how-to-contribute-to-open-source-2022/).
+
+## Get Started
+
+You don’t have to be a developer to contribute, you can even get started by helping us improve this documentation.
+Here's some ideas to get you started:
+
+* Simply fork the FerretDB project you want to contribute to.
+Read the [CONTRIBUTING.md](https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md) and [CODE_OF_CONDUCT.md](https://github.com/FerretDB/FerretDB/blob/main/CODE_OF_CONDUCT.md) files to know more about contributing to the project.
```
Comment (AlekSi) [inline website/docs/contributing.md:25]: The idea was to link to https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md for FerretDB, to https://github.com/FerretDB/dance/blob/main/CONTRIBUTING.md for dance, and to a separate CONTRIBUTING.md in github-actions repo afte FerretDB/github-actions#96

**Review by noisersup — APPROVED** [review]

File: website/docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+More information on contributing to the documentation can be found [here](https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md#contributing-documentation).
+
+## Contributing to the FerretDB repository
+
+* To contribute to this [FerretDB project](https://github.com/FerretDB/FerretDB/), please read the [CONTRIBUTING.md](https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md) and [CODE_OF_CONDUCT.md](https://github.com/FerretDB/FerretDB/blob/main/CODE_OF_CONDUCT.md) guideline to know more.
+* Is everything working as it should?
+If not, please let us know by [creating an issue](https://github.com/FerretDB/FerretDB/issues/new/choose).
+You can create issues for bugs, documentation, features, or enhancements.
+* You can identify issues you would like to work on by looking at the [open issues](https://github.com/FerretDB/FerretDB/issues) for this repository.
+* For first-time contributors looking for issues to work on, kindly check the issues that are labeled [good-first-issues](https://github.com/FerretDB/FerretDB/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
```
Comment (AlekSi) [inline website/docs/contributing.md:37]: I think we should link to https://github.com/FerretDB/FerretDB/contribute instead

File: website/docs/contributing.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+* [github-actions](https://github.com/FerretDB/github-actions): This repository houses our shared GitHub Actions for FerretDB, dance, and other repositories.
+* [dance (stub)](https://github.com/FerretDB/dance): This repository is a FerretDB integration testing tool.
+
+For those taking their first steps in contributing to an open source project.
+Please take a look at this post on [how to contribute to open source software](https://www.ferretdb.io/how-to-contribute-to-open-source-2022/).
+
+## Get Started
+
+You don’t have to be a developer to contribute to FerretDB projects, you can even get started by helping us improve this documentation.
+If you have any questions or suggestions on how we can improve, kindly join our [Slack chat](https://github.com/FerretDB/FerretDB/discussions) or [GitHub discussions](https://join.slack.com/t/ferretdb/shared_invite/zt-zqe9hj8g-ZcMG3~5Cs5u9uuOPnZB8~A).
```
Comment (AlekSi) [inline website/docs/contributing.md:22]: Could we link to `/intro/#community` instead? I'm a bit worried about that Slack URL that should be updated from time to time being in too many places

File: website/docs/contributing.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+* You can identify issues you would like to work on by looking at the [open issues](https://github.com/FerretDB/FerretDB/issues) for this repository.
+* For first-time contributors looking for issues to work on, kindly check the issues that are labeled [good-first-issues](https://github.com/FerretDB/FerretDB/contribute)
+
+## Contributing to dance
+
+* To start contributing to the [dance repository](https://github.com/FerretDB/github-actions) for integration testing, follow the guidelines in the [CONTRIBUTING.md](https://github.com/FerretDB/dance/blob/main/CONTRIBUTING.md) and [CODE_OF_CONDUCT.md](https://github.com/FerretDB/dance/blob/main/CODE_OF_CONDUCT.md) files.
+* Looking for something to work on?
+Check out the [open issues](https://github.com/FerretDB/dance/issues) for this repository.
+
+## Contributing to github-actions
```
Comment (AlekSi) [inline website/docs/contributing.md:45]: Let's link to https://github.com/FerretDB/github-actions/blob/main/CONTRIBUTING.md)

File: website/docs/contributing.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+# How to contribute to FerretDB
+
+FerretDB is an open source project and everyone – developers and non-developers – is welcome to contribute.
+If you’re interested in contributing to FerretDB projects, this documentation will help you get started.
+
+Here are some of the current FerretDB projects that you can fork and contribute to:
+
+* [FerretDB](https://github.com/FerretDB/FerretDB): This repository contains the FerretDB project – an open source alternative project to MongoDB.
+* [github-actions](https://github.com/FerretDB/github-actions): This repository houses our shared GitHub Actions for FerretDB, dance, and other repositories.
+* [dance (stub)](https://github.com/FerretDB/dance): This repository is a FerretDB integration testing tool.
```
Comment (rumyantseva) [inline website/docs/contributing.md:14]:
```suggestion
* [dance](https://github.com/FerretDB/dance): This repository is a FerretDB integration testing tool.
```

File: website/docs/contributing.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+More information on contributing to the documentation can be found [here](https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md#contributing-documentation).
+
+## Contributing to the FerretDB repository
+
+* To contribute to this [FerretDB project](https://github.com/FerretDB/FerretDB/), please read the [CONTRIBUTING.md](https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md) and [CODE_OF_CONDUCT.md](https://github.com/FerretDB/FerretDB/blob/main/CODE_OF_CONDUCT.md) guideline to know more.
+* Is everything working as it should?
+If not, please let us know by [creating an issue](https://github.com/FerretDB/FerretDB/issues/new/choose).
+You can create issues for bugs, documentation, features, or enhancements.
+* You can identify issues you would like to work on by looking at the [open issues](https://github.com/FerretDB/FerretDB/issues) for this repository.
+* For first-time contributors looking for issues to work on, kindly check the issues that are labeled [good-first-issues](https://github.com/FerretDB/FerretDB/contribute)
```
Comment (rumyantseva) [inline website/docs/contributing.md:37]: Maybe something like 

```suggestion
* The most straightforward way to start contributing is to take issues that are labeled [good-first-issues](https://github.com/FerretDB/FerretDB/contribute) 😉
```

I believe, not only first time contributors might want to start from those.

**Review by w84thesun — APPROVED** [review]
LGTM

**Review by rumyantseva — COMMENTED** [review]
LGTM overall, a couple of suggestions (feel free to reject).

File: website/docs/contributing.md (hunk 1 above)
Comment (AlekSi) [reply website/docs/contributing.md:14]: That wasn't done

**Review by AlekSi — CHANGES_REQUESTED** [review]

**Review by rumyantseva — APPROVED** [review]
LGTM

**Review by AlekSi — APPROVED** [review]

**Pull request merged**